learning_resources/views.py

```diff
@@ -520,7 +520,7 @@ class ContentFileViewSet(viewsets.ReadOnlyModelViewSet):
         ContentFile.objects.select_related("run")
         .prefetch_related("content_tags")
         .filter(published=True)
-        .order_by("-updated_on")
+        .order_by("-created_on")
```

Comment:
`created_on` seems more relevant than `updated_on` and is used for the `new` viewset for learning resources too.